.travis/setup_binfmt_misc.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash -ex
+
+echo ':arm:M::\x7fELF\x01\x01\x01\x00\x00\x00\x00\x00\x00\x00\x00\x00\x02\x00\x28\x00:\xff\xff\xff\xff\xff\xff\xff\x00\xff\xff\xff\xff\xff\xff\xff\xff\xfe\xff\xff\xff:/usr/local/bin/qemu-arm-static:' > /proc/sys/fs/binfmt_misc/register
```

Comment:
I think the current runtests won't run as it needs chroot. I've used
proot/chroot so that the binaries are made in a pristine environment.

We need to transfer the RIP binaries to another container and run it from
there so that chroot is not needed.